language/google/cloud/language/entity.py

```diff
@@ -84,7 +80,7 @@ class Entity(object):
     def __init__(self, name, entity_type, metadata, salience, mentions):
         self.name = name
         self.entity_type = entity_type
-        self.wikipedia_url = metadata.pop('wikipedia_url', None)
+        self.wikipedia_url = metadata.get('wikipedia_url', None)
```

Comment:
That's why `pop` falls back to `None`, in case it isn't there. I made it a custom attribute since `wikipedia_url` was the only value empirically observed and since it was there almost all of (all of?) the time.